README.md

```diff
@@ -1,56 +1,47 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Math Magicians
 
+> This is a project for the Math Magicians website for all fans of mathematics. It is a Single Page App (SPA) that allows users to: Make simple calculations, Read a random math-related quote.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+- HTML, CSS , JavaScript
+- SCSS , Webpack, and React
 
-[Live Demo Link](https://livedemo.com)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
 
-To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+To get a local copy up and running follow these simple example steps.
 
```

Comment:
- [ ] Kindly add a way to clone your repo, it wi make it easier for amateur users know how to copy your project and run it locally.